bin/importjs.js

```diff
@@ -1,21 +1,4 @@
 #!/usr/bin/env node
 
-/* eslint-disable strict */
-
-'use strict';
```

Comment:
Do we not actually need this anywhere anymore? I think removing it might end up breaking older node, but maybe that's okay.